saleor/checkout/tests/fixtures/checkout.py

```diff
@@ -7,10 +7,26 @@
 from ....plugins.manager import get_plugins_manager
 from ....product.models import ProductVariantChannelListing
 from ...fetch import fetch_checkout_info, fetch_checkout_lines
-from ...models import Checkout, CheckoutLine, CheckoutMetadata
+from ...models import Checkout, CheckoutLine, CheckoutMetadata, CheckoutShippingMethod
 from ...utils import add_variant_to_checkout
 
 
+@pytest.fixture
+def checkout_shipping_method(shipping_zone):
+    def wrap(checkout):
+        shipping_method = shipping_zone.shipping_methods.first()
```

Comment:
Should we add the possibility to pass `shipping_method` to this function?  `shipping_zone.shipping_methods.first()` should be assigned by default.